end-to-end-tests/src/test/java/tech/pegasys/peeps/consensus/Ibft2ConsensusTest.java

```diff
@@ -13,30 +13,31 @@
 package tech.pegasys.peeps.consensus;
 
 import tech.pegasys.peeps.NetworkTest;
-import tech.pegasys.peeps.NodeConfiguration;
 import tech.pegasys.peeps.SignerConfiguration;
 import tech.pegasys.peeps.network.ConsensusMechanism;
 import tech.pegasys.peeps.network.Network;
 import tech.pegasys.peeps.node.Account;
+import tech.pegasys.peeps.node.Web3Provider;
 import tech.pegasys.peeps.node.model.Hash;
 import tech.pegasys.peeps.node.verification.ValueReceived;
 import tech.pegasys.peeps.node.verification.ValueSent;
 
+import org.apache.tuweni.crypto.SECP256K1.KeyPair;
 import org.apache.tuweni.eth.Address;
 import org.apache.tuweni.units.ethereum.Wei;
 import org.junit.jupiter.api.Test;
 
 public class Ibft2ConsensusTest extends NetworkTest {
 
-  private final NodeConfiguration node = NodeConfiguration.ALPHA;
+  private Web3Provider alphaNode;
   private final SignerConfiguration signer = SignerConfiguration.ALPHA;
 
   @Override
   protected void setUpNetwork(final Network network) {
-    network.addNode(node.id(), node.keys());
-    network.addNode(NodeConfiguration.BETA.id(), NodeConfiguration.BETA.keys());
-    network.set(ConsensusMechanism.IBFT2, node.id());
-    network.addSigner(signer.id(), signer.resources(), node.id());
+    alphaNode = network.addNode("alpha", KeyPair.random());
+    network.addNode("beta", KeyPair.random());
```

Comment:
Why is this 2nd node added if it isn't used?

Reply:
the node is still part of the network, and is used for the "consensus" check at the end.